File: fix.js
const fs = require('fs');
const path = 'd:/Developer/OnriviMarkDown/OnriviMarkDown/frontend/src/components/MarkdownViewer.txx';
let c = fs.readFileSync(path, 'utf8');

c = c.replace(/const isInline = inline \|\| \\(!match && !codeContent\.includes\([\r\n\s]+\)\)\;/, "const isInline = inline || (!match && !codeContent.includes('\\n'));");

fs.writeFileSync(path, c, 'utf8');
console.log('Fixed');

File: fix.py
import re
with open('d:/Developer/OnriviMarkDown/OnriviMarkDown/frontend/src/components/MarkdownViewer.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

fixed = re.sub(
    r"const isInline = inline \|\| \(!match && !codeContent\.includes\('\n\s*'\)\);",
    r"const isInline = inline || (!match && !codeContent.includes('\\n'));",
    content
)

with open('d:/Developer/OnriviMarkDown/OnriviMarkDown/frontend/src/components/MarkdownViewer.tsx', 'w', encoding='utf-8') as f:
    f.write(fixed)
print('Done!')
